.github/workflows/plugins-integration.yml

```diff
@@ -25,14 +25,21 @@ jobs:
           - version: "6.4"
         ansible:
           # https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-core-changelogs
-          - stable-2.12
-          - stable-2.13
-          - stable-2.14
           - stable-2.15
           - stable-2.16
           - devel
         python:
+          - '3.9'
           - '3.10'
+          - '3.11'
+          - '3.12'
```

Comment:
This version of Python is not used anywhere in roles tests. Why?

Reply:
My thought was that the role aren't using raw python, but the modules are.

Reply:
For roles this is the version of Python that is running Ansible code on Ansible controller.
What's your reasoning for adding Python versions 3.9, 3.11 and 3.12 to modules tests (plugins-integration)? why not just replace 3.10 with the latest 3.12? Do we claim support of particular versions of Python anywhere?

Reply:
I think we talk about supporting specific versions of Ansible which in turn ties back to several versions of Python.  I'm not dead set on the change.